dashboard/modules/actor/tests/test_actor.py

```diff
@@ -262,15 +261,16 @@ def actor_table_data_to_dict(message):
             including_default_value_fields=False)
 
     non_state_keys = ("actorId", "jobId", "taskSpec")
+
     for msg in msgs:
         actor_data_dict = actor_table_data_to_dict(msg)
-        # DEPENDENCIES_UNREADY is 0, which would not be keeped in dict. We
-        # need check its original value.
-        if msg.state == 0:
+        # DEPENDENCIES_UNREADY is 0, PENDING_CREATION is 1, which would not be
+        # kept in dict. We need check its original value.
+        if msg.state == 0 or msg.state == 1:
```

Comment:
Actually, 'PENDING_CREATION' will be kept in the dict because it's not the default value.